TrainingGym/env/SumoEnvParallel.py

```diff
@@ -4,10 +4,13 @@
 import sys
 import gym
 from gym import spaces, logger
+from stable_baselines import PPO2
+from stable_baselines.common.callbacks import BaseCallback
 import numpy as np
 import json
 
 # we need to import python modules from the $SUMO_HOME/tools directory
+# TODO: This line isn't necessary if everyone directly installs the sumo python libraries in their python dist
```

Comment:
Don't need to change anything in this pr , but I think it could also be removed if we use venvs